yt_dlp/extractor/nfl.py

```diff
@@ -157,3 +164,131 @@ def _real_extract(self, url):
             'nfl-c-article__title', webpage)) or self._html_search_meta(
             ['og:title', 'twitter:title'], webpage)
         return self.playlist_result(entries, display_id, title)
+
+
+class NFLPlusReplayIE(NFLBaseIE):
+    IE_NAME = 'nfl.com:plus:replay'
+    _VALID_URL = r'https?://(?:www\.)?nfl.com/plus/games/[\w-]+/(?P<id>\d+)'
+    _TESTS = [{
+        'url': 'https://www.nfl.com/plus/games/giants-at-vikings-2022-post-1/1572108',
+        'info_dict': {
+            'id': '1572108',
+            'ext': 'mp4',
+            'title': 'New York Giants at Minnesota Vikings',
+            'description': 'New York Giants play the Minnesota Vikings at U.S. Bank Stadium on January 15, 2023',
+            'uploader': 'NFL',
+            'upload_date': '20230116',
+            'timestamp': 1673864520,
+            'duration': 7157,
+            'categories': ['Game Highlights'],
+            'tags': ['Minnesota Vikings', 'New York Giants', 'Minnesota Vikings vs. New York Giants'],
+            'thumbnail': r're:^https?://.*\.jpg',
+        },
+        'params': {'skip_download': 'm3u8'},
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        return self.url_result(f'{self._ANVATO_PREFIX}{video_id}', AnvatoIE, video_id)
+
+
+class NFLPlusEpisodeIE(NFLBaseIE):
+    IE_NAME = 'nfl.com:plus:episode'
+    _VALID_URL = r'https?://(?:www\.)?nfl.com/plus/episodes/(?P<id>[\w-]+)'
+    _TESTS = [{
+        'note': 'premium content',
+        'url': 'https://www.nfl.com/plus/episodes/kurt-s-qb-insider-conference-championships',
+        'info_dict': {
+            'id': '1576832',
+            'ext': 'mp4',
+            'title': 'Kurt\'s QB Insider: Conference Championships',
+            'description': 'md5:944f7fab56f7a37430bf8473f5473857',
+            'uploader': 'NFL',
+            'upload_date': '20230127',
+            'timestamp': 1674782760,
+            'duration': 730,
+            'categories': ['Analysis'],
+            'tags': ['Cincinnati Bengals at Kansas City Chiefs (2022-POST-3)'],
+            'thumbnail': r're:^https?://.*\.jpg',
+        },
+        'params': {'skip_download': 'm3u8'},
+    }]
+
+    _CLIENT_DATA = {
+        'networkType': 'other',
+        'nflClaimGroupsToAdd': [],
+        'nflClaimGroupsToRemove': [],
+        'clientKey': '4cFUW6DmwJpzT9L7LrG3qRAcABG5s04g',
+        'clientSecret': 'CZuvCL49d9OwfGsR',
+        'deviceId': str(uuid.uuid4()),
+        'deviceInfo': base64.b64encode(json.dumps({
+            'model': 'desktop',
+            'version': 'Chrome',
+            'osName': 'Windows',
+            'osVersion': '10.0',
+        }, separators=(',', ':')).encode()).decode(),
+    }
+    _ACCOUNT_INFO = {}
+    _API_KEY = None
+
+    _TOKEN = None
+    _TOKEN_EXPIRY = 0
+
+    def _get_account_info(self, url, video_id):
+        cookies = self._get_cookies('https://www.nfl.com/')
+        login_token = try_call(lambda: cookies[f'glt_{self._API_KEY}'].value)
```

Comment:
There can be multiple `glt_`... cookies, and they don't always match the API key. In my testing, with each login the the `glt` cookies have had identical values, so any `glt` cookie's value will most likely work. But I figure if there *is* a `glt_{_API_KEY}` cookie, then that is the cookie most likely to work should the values differ for any reason. That's why I had the for loop

Reply:
So this?
```suggestion
        login_token = try_get(
            traverse_obj(cookies, f'glt_{self._API_KEY}', lambda k, _: k.startswith('glt_'), get_all=False),
            lambda x: x.value)
```

If we implement https://discord.com/channels/807245652072857610/1022543137698230362/1074581895918137354, it can further simplify to:


```suggestion
        login_token = traverse_obj(cookies, (
            (f'glt_{self._API_KEY}', lambda k, _: k.startswith('glt_')), 'value'), get_all=False))
```

Reply:
For now, we could do this too, which is cleaner imo than adding `try_get` into the mix:
```suggestion
        login_token = traverse_obj(cookies, ((
            (f'glt_{self._API_KEY}', lambda k, _: k.startswith('glt_')),
            {lambda x: x.value}), get_all=False)
```
.
However, in my testing today, the site set a `gig_loginToken`-prefixed cookie, so I think we should also handle that in case there is no `glt`-prefixed cookie:
```suggestion
        login_token = traverse_obj(cookies, ((
            f'glt_{self._API_KEY}', f'gig_loginToken_{self._API_KEY}',
            lambda k, _: k.startswith('glt_') or k.startswith('gig_loginToken_')),
            {lambda x: x.value}), get_all=False)
```

Reply:
lgtm